controller/kubernetes_configmap_controller.go

```diff
@@ -234,3 +245,26 @@ func (kc *KubernetesConfigMapController) enqueueConfigMapChange(obj interface{})
 
 	kc.queue.Add(key)
 }
+
+func (kc *KubernetesConfigMapController) enqueueConfigMapForStorageClassChange(obj interface{}) {
+	kc.queue.Add(kc.namespace + "/" + types.DefaultStorageClassConfigMapName)
+}
+
+func isLonghornStorageClass(obj interface{}) bool {
+	sc, isSC := obj.(*storagev1.StorageClass)
+	if !isSC {
+		deletedState, ok := obj.(cache.DeletedFinalStateUnknown)
+		if !ok {
+			utilruntime.HandleError(fmt.Errorf("received unexpected obj: %#v", obj))
+			return false
+		}
+
+		sc, ok = deletedState.Obj.(*storagev1.StorageClass)
+		if !ok {
+			utilruntime.HandleError(fmt.Errorf("DeletedFinalStateUnknown contained non StorageClass object: %#v", deletedState.Obj))
+			return false
+		}
+	}
+
+	return sc.Provisioner == types.LonghornDriverName
```

Comment:
Can we filter out other`driver.longhorn.io` SC and only consider the `longhorn` SC with `driver.longhorn.io`. e.g:
```
return sc.Provisioner == types.LonghornDriverName && sc.Name == type.DefaultStorageClassName
```